### DISCUSSION TITLE
Newb LXC setup question for Paperless-ngx

### DISCUSSION THREAD

**Discussion by RobbyRash** [tteck/Proxmox#3543] [Help Needed] [answered]
Apologies but I have read the instructions and every discussion post and can't find a single reference to this.
The instructions say "To Update Paperless-ngx or Show Login Credentials, run the command below (or type update) in the LXC Console."
When I do that, it just says command not found.  How do I actually get the credentials after installing?

**Comment by tteck** [comment, marked as answer]
If in fact you're executing `update` within the LXC console and not the Proxmox shell, it would indicate the script didn't complete successfully.
It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Reply by RobbyRash** [reply]
Got it working now.  Thanks!

**Reply by tteck** [reply]
What was the issue?

**Comment by RobbyRash** [comment]
Operator head space…I actually read the instructions.  Was using the wrong console.
